DbConnection/src/main/java/com/solvd/laba/db/dao/GenericDAO.java

```diff
@@ -0,0 +1,11 @@
+package com.solvd.laba.db.dao;
```

Comment:
Is it possible to just put the abstract methods in this interface inside the abstract class and just delete this interface?

Reply:
I made some changes. Now my interface includes some methods and are implemented by all the DAO classes. The abstract class provides commonly functionality to all the dao classes.